tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -2286,6 +2285,29 @@ bool DeclResultIdMapper::finalizeStageIOLocations(bool forInput) {
   return assignLocations(vars, nextLocs, &stageVariableLocationInfo);
 }
 
+llvm::DenseMap<const SpirvFunction *, SmallVector<StageVar, 8>>
+DeclResultIdMapper::GetStageVarsPerFunction() {
```

Comment:
nit: I believe member functions begin with lowercase letters (ie, `getStageVarsPerFunction`) in the rest of our codebase